vendor/modules.txt

```diff
@@ -0,0 +1,322 @@
+# cloud.google.com/go v0.34.0
```

Comment:
Is this file coming with dep?

Reply:
Let me vendor it again

Reply:
Yes, tried to remove the vendor and did `go mod tidy && go mod vendor`